projects/contributor-experience.md

```diff
@@ -0,0 +1,115 @@
+only in adoption and lines of code, but also in the number of individuals who
+contribute to this project in different ways.
+
+While in the beginning of the project contributors knew each other well and
+could share implicit in-group knowledge effortlessly, the project has outgrown that
+phase a while ago and we see different kinds of problems, that make it hard
+for long-term contributors to maintain the project sustainably and for new-comers
+to enter the project effortlessly.
+
+In this project proposal we suggest to form a cross-cutting special interest group
+that takes responsibility for driving contributor experience projects and that
+closely collaborates with all other SIGs and governing bodies to increase contributor
+sustainability, velocity and happiness.
+
+### Current challenges
+
+Among others the challenges the OpenTelemetry project is facing in regards of
+contributor experience are:
+
+* No consistent way of moving contributors "up the ladder", e.g. from member to
+  approver, from approver to maintainer etc, which makes those changes invisible
+  to the larger project and opaque to project leadership on who is doing
+  what.
+* No clear onboarding path for new contributors, or contributors taking on new
+  responsibilities within the project (triagers, approvers, maintainers, committee
+  members), which leads to friction in understanding of the overall project goals,
+  to misalignment in implementations and the feeling of contributors being remote
+  to other parts of the project.
+* No explicit contributor guidebook that helps contributors to answer common
+  questions no matter where they are on the contributors' ladder.
+* No mentorship available for new contributors or contributors stepping up into new roles to
+  make them part of the project and aware of expectations.  
+* Only implicit and unstructured recognition for contributors, instead of explicit
+   recognition which would help contributors to feel seen and recognized.
+* No structured collaboration across SIGs, especially in the case of cross-cutting
+  concerns (Security, Docs, End-User, ...), leading to information silos and misalignment.
+* SIGs are short of contributors and require some help to advertise their need
+  (blog post, social posts, etc) and to lower the barrier of entry into their
+  sub-project.
+* _add more_
+
+Note that there are individuals within the project who try to tackle some of those
+aspects as individuals and not in a team effort, notable:
+
+* [Add skeleton for maintainer and contributor guidebooks](https://github.com/open-telemetry/community/pull/2051)
+* Mentorship efforts, e.g. [Outreachy](https://cloud-native.slack.com/archives/C060GFUL0P6)
+* Surveys by the End User SIG that aim towards contributor experience
+* _add more_
+
+Also related to this initiative is the [Check-In process of the Governance Committee](https://github.com/open-telemetry/community/blob/main/gc-check-ins.md)
```

Comment:
Another related resource is the CNCF's cross-cutting Contributor Experience SIG.  https://github.com/kubernetes/community/tree/master/sig-contributor-experience

Feels like one of the things this group might do is maintain a relationship with them and consult them for best practices and resources where that makes sense, as well as upstream our governance resources into their repos for other projects when that makes sense.

Reply:
How do we feel all this fits with the current role of the "community manager"? Maybe that was an interim solution and we need a ContribEx SIG, not a single community manager, but if we're going to continue that role we should address the relationship. cc @austinlparker

Reply:
> Feels like one of the things this group might do is maintain a relationship with them and consult them for best practices and resources where that makes sense, as well as upstream our governance resources into their repos for other projects when that makes sense.

Ah, let me another item for that :-)

Good point around the Community Manager role, @austinlparker wdyt?

Reply:
I'm not sure I agree with that. I think the Community Manager handles things like event planning, which I don't see as part of this SIG.

Reply:
@jpkrohling - well, it's possible that the answer really is that ContribEx doesn't overlap with community management.

My sense based on what you've said and what I already think, is that the distinction is that ContribEx is focused on the internal contributor community, and the community manager is focused on how OpenTelemetry as a project shows up in developer communities external to OpenTelemetry.

But I think that's worth articulating clearly even if we're writing down the obvious, because "community manager" could be a reasonable job title for either of those focuses.

Reply:
I think this can be a good thing to add to the charter. I'm marking this as resolved for now.